archive/imo1988_q6.lean

```diff
@@ -212,7 +212,7 @@ begin
     apply ne_of_lt,
     calc x*x + x*x = x*x * 2       : by rw mul_two
                ... ≤ x*x * k       : nat.mul_le_mul_left (x*x) k_lt_one
-               ... < (x*x + 1) * k : by apply mul_lt_mul; linarith },
+               ... < (x*x + 1) * k : by apply mul_lt_mul; nlinarith },
```

Comment:
```suggestion
               ... < (x*x + 1) * k : by linarith },
```
It seems this works?

Reply:
Ah, nice. That's thanks to the change in #2995.